src/nrnpython/nrnpython.h

```diff
@@ -23,30 +23,18 @@
 #endif
 #include <nrnwrap_Python.h>
 
-#if (PY_MAJOR_VERSION >= 3)
 #define myPyMODINIT_FUNC PyObject *
```

Comment:
I have no real problem with this. However there is longer any variant. So every occurrence could be replaced with PyObject*. (And doing it here begs the question why the ones below still exist). To me this is not a merge stopper.